src/interaction/interactiveTarget.js

```diff
@@ -29,7 +31,12 @@ module.exports = {
     /**
      * @todo Needs docs.
      */
-    defaultCursor: 'pointer',
+    defaultCursor: 'pointer',    
+    
+    /**
+     * The threshold in px used to check if a tap occured on the interactive object.
+     */
+    tapThreshold: 8,
```

Comment:
Experience.. feeling.. we make a lot of apps for kids, who always move their fingers before releasing, but are intending making a tap. So only 2px is to little.